package.json

```diff
@@ -1,10 +1,34 @@
 {
-  "private": true,
+  "name": "react-app-rewired",
+  "version": "2.0.3",
+  "description": "Tweak the create-react-app webpack config(s) without using 'eject' and without creating a fork of the react-scripts",
+  "private": false,
+  "bin": {
+    "react-app-rewired": "./bin/index.js"
+  },
   "scripts": {
-    "postinstall": "lerna bootstrap",
-    "publish": "lerna publish && lerna-npm-readme-hack"
+    "build": "yarn --cwd 'test/react-app' build",
+    "clean": "rm -rf node_modules",
+    "setup": "yarn deps && yarn link && yarn --cwd 'test/react-app' link 'react-app-rewired'",
+    "start": "yarn --cwd 'test/react-app' start",
+    "teardown": "yarn unlink && yarn clean && yarn --cwd 'test/react-app' clean",
+    "test": "yarn --cwd 'test/react-app' test",
+    "deps": "yarn && yarn --cwd 'test/react-app'"
+  },
+  "peerDependencies": {
+    "react-scripts": "^2.1.3"
+  },
+  "dependencies": {
+    "cross-spawn": "^6.0.5",
+    "dotenv": "^6.2.0",
+    "semver": "^5.6.0"
   },
   "devDependencies": {
-    "lerna": "2.0.0"
-  }
+    "react-scripts": "^2.1.3"
```

Comment:
`react-scripts` required as a `devDependency` here so that we can link in `test/react-app`